x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/DataFrameSurvivesUpgradeIT.java

```diff
@@ -113,35 +122,44 @@ private void cleanUpTransforms() throws Exception {
 
     private void createAndStartContinuousDataFrame() throws Exception {
         createIndex(CONTINUOUS_DATA_FRAME_SOURCE);
-        long totalDocsWritten = 0;
+        long totalDocsWrittenSum = 0;
         for (TimeValue bucket : BUCKETS) {
             int docs = randomIntBetween(1, 25);
             putData(CONTINUOUS_DATA_FRAME_SOURCE, docs, bucket, ENTITIES);
-            totalDocsWritten += docs * ENTITIES.size();
+            totalDocsWrittenSum += docs * ENTITIES.size();
         }
-
+        long totalDocsWritten = totalDocsWrittenSum;
```

Comment:
nit: if you use `totalDocsWrittenSum` on line 138, you do not need this extra variable